interactive_ai/tests/e2e/Makefile

```diff
@@ -0,0 +1,72 @@
+include ../../../Makefile.shared-python
+
+MAKEFILE_DIR := $(shell pwd)
+OPENAPI_YAML_SPEC := ../../../docs/rest_api/openapi_internal.yaml
+OPENAPI_JSON_SPEC := /tmp/openapi_internal.json
+CLIENT_NAME := geti_client
+CLIENT_PATH := rest_client
+SPEC_HASH_FILE := .openapi_spec_hash
+
+.PHONY: all check_yaml_spec generate_json_spec generate_client generate_client_if_needed test test_wip test_smoke
+
+all: test
+
+bdd-venv:
+	uv sync
+	$(MAKE) generate_client_if_needed;
+	uv pip install -e $(CLIENT_PATH)
+
+check_yaml_spec:
+	@if [ ! -f $(OPENAPI_YAML_SPEC) ]; then \
+		echo "Error: OpenAPI YAML spec file not found at $(OPENAPI_YAML_SPEC)"; \
+		exit 1; \
+	fi
+
+generate_json_spec: check_yaml_spec
+	@command -v swagger-cli >/dev/null 2>&1 || { echo >&2 "swagger-cli is not installed. Aborting."; exit 1; }
+	swagger-cli bundle $(OPENAPI_YAML_SPEC) -o $(OPENAPI_JSON_SPEC) --dereference
```

Comment:
Have we considered upgrading to [Redocly CLI](https://redocly.com/docs/cli)? It should be fully compatible with `swagger-cli`.